### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by Silenc3IsGold3n** [MohrJonas/icpi#1]
You put a decent amount of time into this post:
https://github.com/Vanilla-OS/apx/issues/226#issue-1732910362

And to be honest it looks like a decent alternative to apx.

Might I suggest moving a lot of the content from that post into a readme for this project?

**Comment by MohrJonas** [comment]
Yeah I can do that, but I'm not sure if this project is still relevant now that APX v.2 is right around the corner
